src/utils/renderStats.js

```diff
@@ -0,0 +1,60 @@
+import _ from "lodash";
+
+class RenderStats {
+  constructor({ g, model }) {
+    this.g = g;
+    this.model = model;
+
+    this.clear();
+  }
+
+  clear() {
+    this.stats = {
+      renderedSequenceBlock: { },
```

Comment:
This holds information for just the rendered sequence blocks.
If QA needs information for all the sequence blocks, we can a `sequenceBlock` field later.